packages/components/src/BrandMoment/BrandMoment.tsx

```diff
@@ -10,7 +10,9 @@ import { useMediaQueries } from "~utils/useMediaQueries"
 import styles from "./BrandMoment.module.scss"
 
 export type BrandMomentProps = {
-  mood: "informative" | "positive" | "negative"
+  // @deprecated use `color` instead
+  mood?: "informative" | "positive" | "negative"
+  color: "blue" | "green" | "red"
```

Comment:
Making this required makes it a breaking change 😅 You might need a discriminated union to enforce one or the other as required

Reply:
Oh I got this the other way round, maybe make `color` optional instead?